### ISSUE TITLE
LutaML-Model integration in Plurimath

### ISSUE THREAD

**Issue by suleman-uzair** [plurimath/plurimath#301]
Parsing of **MathML** and **OMML** in **Plurimath** using **LutaML-Model** gem.

- [ ] MathML parsing and transformation
- [ ] OMML parsing and transformation

**Comment by suleman-uzair** [comment]
@ronaldtse I’m working on the integration of **LutaML-Model** in **Plurimath MathML** parsing.
After parsing **XML** using **LutaML-Model**, we will convert it to **Plurimath::Math::Formula** directly and remove the existing **Parslet** functionality for transformation(from parsed **XML** nodes to **Formula**).

Are we going in the right direction?

**Comment by ronaldtse** [comment]
Yes, we will use Lutaml::Model XML parsing functionality, and obsolete the parsing functionality in Plurimath for MathML and OMML.

The XML Engine functionality in Plurimath is well done: can you or @HassanAkbar help migrate them to Lutaml::Model? In particular, Lutaml::Model's XML adapters for Oga and Ox don't work properly yet. I think Lutaml::Model's XML adapters is in need of some improvement.

**Comment by suleman-uzair** [comment]
> can you or [@HassanAkbar](https://github.com/HassanAkbar) help migrate them to Lutaml::Model?

@ronaldtse, Sure, I’ll work on it after the current task.

**Comment by suleman-uzair** [comment]
Both tracked tasks are complete: #302 via #304, and #303 via #414. Closing this now.
